src/Components/drills/FrisbeeDrillIllustration.js

```diff
@@ -88,14 +89,26 @@ const FrisbeeDrillIllustration = (props) => {
     );
   };
 
-  const renderPagination = () => {
+  const renderCounter = (index) => {
+    if (props.drill.steps.length !== 1) {
+      return (
+        <View style={styles.counter}>
+          <Text style={styles.textCounter}>
+            {index + 1}/{props.drill.steps.length}
+          </Text>
+        </View>
+      );
+    }
+  };
+
+  const renderPagination = (index) => {
     return (
       <Pagination
         dotsLength={props.drill.steps.length}
-        activeDotIndex={activeIndex}
+        activeDotIndex={index}
```

Comment:
This breaks the effect used on line 21 : when you move to another drill in the top navigation, it should display the first step. If you use the index directly, it stays on the step it was displaying